var/spack/repos/builtin/packages/resolve/package.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Resolve(CMakePackage, CudaPackage):
+    """FIXME: Put a proper description of your package here."""
```

Comment:
This is my suggestion:
```suggestion
    """ReSolve is a library of GPU-resident sparse linear solvers. It contains iterative and direct solvers designed to run on NVIDIA and AMD GPUs, as well as CPU devices."""
```
@kswirydo, please chime in.